apps/site/assets/ts/stop/components/amenities/ParkingAmenityCard.tsx

```diff
@@ -159,9 +159,7 @@ const ParkingAmenityCard = ({
   alertsForParking: Alert[];
 }): JSX.Element => {
   const icon = (
-    <span className="m-stop-page__icon u-color-brand-primary">
-      {parkingIcon()}
-    </span>
+    <span className="m-stop-page__icon text-primary">{parkingIcon()}</span>
```

Comment:
**question:** Would this be appropriately overridden in the case where the card has the "disabled" state (where the icon and text are black)?